machine-learning/tutorials/GitHubIssueClassification/GitHubIssueClassification.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
can you target something > 2.0? 2.0 is out of support already

Reply:
2,1 now